packages/host/app/components/operator-mode/overlays.gts

```diff
@@ -37,6 +37,10 @@ export default class OperatorModeOverlays extends Component<Signature> {
     );
   }
 
+  isPillDisplay(renderedCard: RenderedCardForOverlayActions) {
+    return renderedCard.element.classList.contains('pill');
```

Comment:
Is there a more robust way to detect this? Someone could remove the pill class name and then this would be a bug.

Reply:
updated [addc943](https://github.com/cardstack/boxel/pull/672/commits/addc943e135e32b1d17e55c97a88c457d63f0a42)